.github/workflows/ci.yaml

```diff
@@ -140,65 +143,18 @@ jobs:
 
       - name: Run integration tests
         run: |
-          sg snap_microk8s -c "juju add-model knative-test"
-          sg snap_microk8s -c "tox -vve integration -- --model knative-test --charms-path=${{ github.workspace }}/charms"
-
-      - run: kubectl get all -A
-        if: failure()
-
-      - run: kubectl get crds
-        if: failure()
-
-      - run: juju status
-        if: failure()
-
-      - uses: canonical/kubeflow-ci/actions/dump-charm-debug-artifacts@main
-        if: failure()
-
-  integration-observability:
-    name: Observability Integration Test
-    needs:
-      - build
-    runs-on: ubuntu-20.04
-    steps:
-      # Ideally we'd use self-hosted runners, but this effort is still not stable
-      # This action will remove unused software (dotnet, haskell, android libs, codeql,
-      # and docker images) from the GH runner.
-      # This leaves ~45GB free as of 2024-04-10, but this amount has varied as GH changed their
-      # runners
-      - name: Maximise GH runner space
-        uses: jlumbroso/free-disk-space@v1.3.1
+          sg snap_microk8s -c "tox -vve ${{ matrix.environment }} -- --model knative-test --charms-path=${{ github.workspace }}/charms/"
 
-      - name: Check out repo
-        uses: actions/checkout@v4
+      - run: kubectl get pod/prometheus-k8s-0 -n knative-test -o=jsonpath='{.status}'
+        if: failure() && ${{ matrix.environment=='cos-integration' }} 
```

Comment:
do you mean `grafana-agent-k8s-0`? The cos integration tests here don't deploy prometheus iiuc

Reply:
I actually went ahead and removed those two `get` since we have enough logging in the steps below